### PR TITLE
Devdocs 3983 add cart line items fixes

### PR DESCRIPTION
**Pull request by markcmurphy**
# [DEVDOCS-3983](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-3983)

## What changed?
* change add cart line items `line_items` to array from object
* some other small changes with data type and examples

### CODE REVIEW

**Review by traci707 — APPROVED** [review]
Looks good Mark. Did I previously structure this incorrectly?

File: reference/carts.v3.yml [hunk 1]
```diff
@@ -183,11 +183,11 @@ paths:
   '/stores/{store_hash}/v3/carts/{cartId}/items':
     post:
       description: |-
-        Adds line item to the *Cart*.
+        Adds one or more line items to the cart.
 
         **Usage Notes**
 
-        To add a custom item use `custom_items`. 
+        To add one or more custom items use `custom_items`.
```
Comment (slsriehl) [inline reference/carts.v3.yml:190]:
```suggestion
        To add one or more custom items, use `custom_items`.
```

File: reference/carts.v3.yml [hunk 2]
```diff
@@ -881,71 +846,21 @@ components:
               type: number
             product_id:
               type: number
-            variant_id:
```
Comment (slsriehl) [inline reference/carts.v3.yml:884]: is this not needed? forgive me if this is the point of the ticket.

File: reference/carts.v3.yml [hunk 3]
```diff
@@ -881,71 +846,21 @@ components:
...
                   option_id:
                     type: number
                   option_value:
                     type: string
-                title: Product Option Selection
           required:
             - quantity
             - product_id
-            - variant_id
-        gift_certificate:
```
Comment (slsriehl) [inline reference/carts.v3.yml:903]: checking in to make sure this section wasn't removed by accident, since I don't think I see it elsewhere.

**Comment by slsriehl** [comment]
@markcmurphy I have one suggestion and a couple questions.

**Pull request closed without merging**